easybuild/easyconfigs/s/segemehl/segemehl-0.2.0-intel-2017b.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
```

Comment:
@boegel - any reason to remove Pablo's credit?

Reply:
Hmm, that's only there in older `segemehl` easyconfigs, not in the `segemehl-0.2.0-foss-2016b.eb` that I used as a base.

I'll add it back, both in this easyconfig & in `segemehl-0.2.0-foss-2016b.eb`, thanks for bringing this up.